src/Services/DBusInterfaces/Device.vala

```diff
@@ -37,10 +37,10 @@ namespace Power.Services.DBusInterfaces {
 
     [DBus (name = "org.freedesktop.UPower.Device")]
     public interface Device : DBusProxy {
-        public abstract HistoryDataPoint[] GetHistory (string type, uint32 timespan, uint32 resolution) throws GLib.Error;
+        public abstract HistoryDataPoint[] GetHistory (string type, uint32 timespan, uint32 resolution) throws GLib.Error; // vala-lint=naming-convention
```

Comment:
This would work for D-Bus
```suggestion
        public abstract HistoryDataPoint[] get_history (string type, uint32 timespan, uint32 resolution) throws GLib.Error;
```